recipe/build_base.sh

```diff
@@ -19,4 +19,4 @@ sed -i.bak "s|/usr/local|${PREFIX}|" setupext.py
 
 export MPLSETUPCFG=conda_mpl_config.cfg
 
-$PYTHON -m pip install . --no-deps -vv
+$PYTHON -m pip install --no-deps -vv .
```

Comment:
```suggestion
$PYTHON -m pip install --no-deps -vv . --no-build-isolation
```

or the "externally managed" (which I do not know off the top of my head) flag?

The arm jobs looks like they are trying to install numpy from source.

Reply:
Yeah, I tried that once but that location after the "." failed, so I went back to incrementally get there.

Still an open question why that would even been needed seeing as how conda-forge's numpy recipe builds off a pyproject.toml and doesn't need this option.